omnisolver/pt/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Package with purely random reference sampler.
```

Comment:
Is this docstring correct? It hsould say something about parallel tempering.

Reply:
Fixed in 58116e7